### DISCUSSION TITLE
Update on zeebe-client-rust

### DISCUSSION THREAD

**Discussion by pihme** [camunda-community-hub/community#35] [Share your work]
Hi all, just wanted to share an update on https://github.com/camunda-community-hub/zeebe-client-rust project.

We made some progress during Camunda’s internal Summer Hack Days:

* Added CLI for all Zeebe Client commands (as of version 8.0.0)
* Added support for OAuth authentication, which means you can connect to a cluster running in Camunda Cloud

We still need want to review the code we produce during the hack days. For all brace early adopters, you can already try it out here:
https://github.com/camunda-community-hub/zeebe-client-rust/tree/hackdays

Also, we would love to hear from you if you are interested in this project.

**Comment by xomiamoore** [comment]
Thanks for sharing @pihme!
